### DISCUSSION TITLE
Is there a plan to creare hv-uikit for Angular ?

### DISCUSSION THREAD

**Discussion by WhiteWalker1** [lumada-design/hv-uikit-react#4312] [General]
We are developing applications for Web and Local desktop applications using electron js and angular. We are wondering if there will be a plan to generate a similar version of this UI kit for angular applications.
Thanks, Julian.

**Comment by zettca** [comment]
Hey @WhiteWalker1

There are no plans to support Angular.

The UI Kit did have an initial Angular implementation (very long ago, circa 2019), but it was dropped as it wasn't worth the maintenance cost & effort.

**Reply by WhiteWalker1** [reply]
Thanks Bruno, appreciate your answer.
